test/pjrt/test_torchrun.py

```diff
@@ -28,6 +28,41 @@ def test_all_gather(self):
 
     expected = torch.arange(0, expected_world_size, step=1, dtype=torch.float32)
     torch.testing.assert_close(result.cpu(), expected)
+    dist.destroy_process_group()
+
+  def test_all_reduce(self):
+    # The test is inspired by https://pytorch.org/docs/stable/distributed.html#torch.distributed.all_reduce
+    dist.init_process_group('xla', init_method='xla://')
```

Comment:
Can we init/destroy the process group in setUpTest/tearDownTest to cover the early exit case?

Reply:
done